app/controllers/api/v1/admin/trends/tags_controller.rb

```diff
@@ -11,8 +11,36 @@ def index
     end
   end
 
+  def update
+    raise Mastodon::NotPermittedError unless current_user&.can?(:manage_taxonomies)
+
+    tag = Tag.find(params[:id])
+    tag.update(tag_params.merge(reviewed_at: Time.now.utc))
+    render json: tag, serializer: REST::Admin::TagSerializer
+  end
```

Comment:
I'm not sure this is the right place, e.g. changing a tag's case does not really have anything to do with trends.

Another issue with this bit of code is that there is no error handling.

Reply:
This mirrors the same functionality in the admin ui at `/admin/tags/:id` (when you click on a tag from the list at `/admin/trends/tags`)

Reply:
I know, but just like there's an `Api::V1::TagsController`, it may make sense to have `Api::V1::Admin::TagsController` that is not tied to trends.